validator/client/runner.go

```diff
@@ -66,24 +67,32 @@ func run(ctx context.Context, v Validator) {
 				handleAssignmentError(err, slot)
 				continue
 			}
-			role := v.RoleAt(slot)
+			var wg sync.WaitGroup
```

Comment:
any reason to use a wait group here  ? after exiting from performing their roles the validators do not do anything else. If one validator is blocked in performing its role then all the validators will be held up and might skip the next slot.

Reply:
maybe we should have an indicator if the validators task take more than a slot time. I think we should finish all tasks that was assigned before proceeding

Reply:
Please remove the waitgroup

Reply:
done